client/package-lock.json

```diff
@@ -1295,11 +1293,6 @@
       "resolved": "https://registry.npmjs.org/@csstools/normalize.css/-/normalize.css-10.1.0.tgz",
       "integrity": "sha512-ij4wRiunFfaJxjB0BdrYHIH8FxBJpOwNPhhAcunlmPdXudL1WQV1qoP9un6JsEBAgQH+7UXyyjh0g7jTxXK6tg=="
     },
-    "node_modules/@emotion/hash": {
```

Comment:
Some packages are deleted here. Might be important to have them as it'll break other's code that uses these packages.

Reply:
should i just revert the changes on the package then?